nltk/corpus/reader/wordnet.py

```diff
@@ -1203,59 +1208,75 @@ def __init__(self, root, omw_reader):
         # load the exception file data into memory
         self._load_exception_map()
 
+        self.nomap = []
+        self.splits = {}
+
         # map from WordNet 3.0 for OMW data
         self.map30 = self.map_wn30()
 
         # Language data attributes
         self.lg_attrs = ["lemma", "none", "def", "exe"]
 
-    def corpus2sk(self, corpus=None):
+    def indexSense(self, version=None):
```

Comment:
```suggestion
    def index_sense(self, version=None):
```
Could we stick with snake case method names, as suggested by the Python [style guide](https://peps.python.org/pep-0008/#method-names-and-instance-variables). Optionally, we can use a single underscore in front of a method name if it's not meant to be used by the public. This may be the case for e.g. `mapToMany`?

Reply:
Thanks @tomaarsen, I have added all your suggestions for the function names, and also changed the variable names.